src/task_7_1.py

```diff
@@ -0,0 +1,54 @@
+'''1. Написать 12 функций :
+1. Дюймы в сантиметры 2. Сантиметры в дюймы 3. Мили в километры
+4. Километры в мили   5. Фунты в килограммы 6. Килограммы в фунты
+7. Унции в граммы     8. Граммы в унции     9. Галлон в литры
+10. Литры в галлоны   11. Пинты в литры     12. Литры в пинты'''
+
+def inch_cm(value: float) -> float:
+    # Дюймы в сантиметры
```

Comment:
Лучше писать не комментарий а докстрингу, и так для каждой функции
```suggestion
    """Дюймы в сантиметры"""
```